articles/connections/passing-parameters-to-idps.md

```diff
@@ -0,0 +1,127 @@
+# Passing Parameters to Identity Providers
+
+For Identity Providers that support parameters, you can pass provider-specific parameters to the Authorization endpoint by configuring either static values per connection or dynamic values per user.
+
+Only [valid OAuth 2.0/OIDC parameters](http://openid.net/specs/openid-connect-core-1_0.html#AuthorizationEndpoint) are accepted.
+
+Some Identity Providers do not support parameters. Check your desired Identity Provider's documentation to see if parameters are supported.
+
+## Passing Static Parameters
+
+To pass static parameters per connection, you can use the `upstream_params` element of the `options` attribute when configuring a connection via the Management API. Each time your connection is used, the specified parameters will be sent to the Identity Provider.
+
+As an example, let's use WordPress, which allows you to pass an optional `blog` parameter to its OAuth 2.0 authorization endpoint (for more information, see [WordPress's OAuth 2.0 documentation](https://developer.wordpress.com/docs/oauth2/)). Let's assume that you want to always request that users have access to the `myblog.wordpress.com` blog when logging in using WordPress.
+
+To accomplish this, you will map WordPress's `blog` parameter to the existing accepted parameter of `access_type` and assign it a default value of `myblog.wordpress.com`:
+
+```
+{
+  "name": "WordPressConn",
+  "strategy": "wordpress",
+  "options": {
+    "upstream_params": {
+      "blog": {
+        "alias": "access_type",
+        "value": "myblog.wordpress.com"
+      }
+    }
+  }
+}
+```
+
+```har
+{
+  "method": "PATCH",
+  "url": "https://${account.namespace}/api/v2/connections/CONNECTION-ID",
+  "headers": [
+    { "name": "Content-Type", "value": "application/json" }
+  ],
+  "postData": {
+    "mimeType": "application/json",
+    "text": "{ 
+      \"options\": { 
+        \"client_id\": \"...\", 
+        \"client_secret\": \"...\", 
+        \"upstream_params\": { 
+          \"blog\": { 
+            \"alias\": \"access_type\",
+            \"value\": \"myblog.wordpress.com\"
+          }
+        }, 
+        \"authorizationURL\": \"https://public-api.wordpress.com/oauth2/authorize\", 
+        \"tokenURL\": \"https://public-api.wordpress.com/oauth2/token\",
+        \"scope\": \"auth\" 
+      } 
+    }"
+  }
+}
+```
+
+You can test out your code and see other available attributes in our [Management API Explorer](https://auth0.com/docs/api/management/v2#!/Connections/post_connections).
```

Comment:
Always use relative URLs for internal auth0.com/docs links. For example, if the absolute path to the document is `https://auth0.com/docs/identityproviders`, use `/identityproviders`. These links will be correctly formatted in the build process.

So:
`https://auth0.com/docs/api/management/v2#!/Connections/post_connections` -> `/management/v2#!/Connections/post_connections`